src/components/CodeManagement/index.jsx

```diff
@@ -49,6 +49,11 @@ class CodeManagement extends React.Component {
         state: {},
       });
     }
+
+    // To fetch email templates for assign, remind, and revoke
+    this.props.fetchEmailTemplates({
```

Comment:
What is the purpose of active here? Can you please add the comment in code for it.